components/nav-list/src/index.scss

```diff
@@ -27,6 +27,10 @@
   justify-content: start;
 }
 
+.utrecht-nav-list--inline {
+  inline-size: auto;
```

Comment:
Bij `--inline` zou ik eerder `display: inline-flex` verwachten dan dit, maar ik weet niet of dat een vergelijkbaar effect heeft. Wat denk jij?